### PR TITLE
Update README with hot deploy

### PR DESCRIPTION
**Pull request by jhadvig**
@bparees PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,5 +84,16 @@ It will also be necessary to update your application to talk to your database ba
 
 You will then need to rebuild the application.  This is done via either a `oc start-build` command, or through the web console, or a webhook trigger in github initiating a build after the code changes are pushed.
 
+### Hot Deploy
+
+Hot deploy works in the built cakaphp image out of the box.
```
Comment (bparees) [inline README.md]: "Hot deploy works out of the box in the php image used with this example."

File: README.md [hunk 2]
```diff
@@ -84,5 +84,16 @@ It will also be necessary to update your application to talk to your database ba
...
+
+To change your source code in the running container you need first [oc exec](https://docs.openshift.org/latest/cli_reference/basic_cli_operations.html#troubleshooting-and-debugging-cli-operations) into it.
```
Comment (bparees) [inline README.md]: s/you need first/you need to/

File: README.md [hunk 3]
```diff
@@ -84,5 +84,16 @@ It will also be necessary to update your application to talk to your database ba
...
+```
+oc exec -p <POD_ID> -c <CONTAINER_ID> -i -t -- bash -il
```
Comment (bparees) [inline README.md]: "oc rsh" is easier.

**Comment by jhadvig** [comment]
@bparees updated. I've also sent a PR which adds the `oc rsh` into our docs, since it wasn't there, but kept the link to it, so once its merged we dont have to update it.
https://github.com/openshift/openshift-docs/pull/977

**Comment by bparees** [comment]
lgtm

**Pull request merged**